notesapi/v1/serializers.py

```diff
@@ -0,0 +1,74 @@
+"""
+Serializers for Notes API.
+"""
+
+import json
+from rest_framework import serializers
+from notesapi.v1.models import Note
+
+
+class NoteSerializer(serializers.ModelSerializer):
+    """
+    Student Notes Model Serializer.
+    """
+    class Meta(object):
+        """
+        Model Serializer Meta Class.
+        """
+        model = Note
+        exclude = ('user_id',)
+
+    id = serializers.CharField(source='pk')
+    user = serializers.CharField(source='user_id')
+    ranges = serializers.SerializerMethodField()
+    tags = serializers.SerializerMethodField()
+
+    def get_ranges(self, note):
+        """
+        Return note ranges.
+        """
+        return json.loads(note.ranges)
+
+    def get_tags(self, note):
+        """
+        Return note tags.
+        """
+        return json.loads(note.tags)
+
+
+class NotesElasticSearchSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Student Notes Elastic Search Serializer.
+    """
+    id = serializers.CharField(source='pk')
+    user = serializers.CharField(source='user_id')
```

Comment:
I note in the logic in the view, that this replaces, `user` is not being picked from `user_id`. Can you check if we need to do this or will the elastic search responses only have the `user` field?